docs/DeveloperGuide.md

```diff
@@ -290,6 +290,43 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
    * 4a1. System displays an error message.
 
   Use case ends.
+ 
+**Use case: Edit a patient**
+
+**MSS**
+
+1. User requests to edit a patient.
+2. User inputs the new information about the patient.
+3. System edits the patient information to the new information.
+4. System saves the new patient information.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. System realises that no optional fields are inputted.
+
```

Comment:
If I'm not wrong the textbook didn't have these white spaces.

Reply:
whoops, let me rectify that